PPTX-to-Image-conversion/Convert-PowerPoint-presentation-to-Image/Azure/Azure_Functions/Console_Application/PowerPoint-Presentation-to-Image/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace ConvertPresentationtoImage

```

Comment:
Use proper sample naming everywhere in this sample

Reply:
Updated